cloud2sql/parquet.py

```diff
@@ -12,22 +12,35 @@
 import pyarrow.parquet as pq
 from pathlib import Path
 from dataclasses import dataclass
+import dataclasses
 from abc import ABC
+import json
 
 
 class ArrowModel:
-    def __init__(self, model: Model):
+    def __init__(self, model: Model, output_format: Literal["parquet", "csv"]):
         self.model = model
         self.table_kinds = [
             kind
             for kind in model.kinds.values()
             if kind.aggregate_root and kind.runtime_kind is None and kind.fqn not in base_kinds
         ]
         self.schemas: Dict[str, pa.Schema] = {}
+        self.output_format: Literal["parquet", "csv"] = output_format
 
     def _pyarrow_type(self, kind: str) -> pa.lib.DataType:
-        if kind.startswith("dict") or "[]" in kind:
-            return pa.string()  # dicts and lists are converted to json strings
+        if "[]" in kind:
+            #  csv does not support lists
+            if self.output_format == "csv":
+                return pa.string()
```

Comment:
Instead of making the output format part of the object with if/then logic here, I would recommend getting a specific model for csv and one for parquet.